src/runMocha.js

```diff
@@ -9,18 +9,21 @@ const runMocha = ({ config, testPath, globalConfig }, workerCallback) => {
   class Reporter extends Mocha.reporters.Base {
     constructor(runner) {
       super(runner);
-      const tests = [];
-      const pending = [];
-      const failures = [];
-      const passes = [];
+      const tests = new Set();
```

Comment:
Mocha 3 works down to node versions that lack Set entirely; if we’re going to use it, let’s be sure to be using a polyfill.

Reply:
Gadzooks, mocha 3 supports all the way back to node 0.10.

My personal preference would be to either:

- Revert this last commit and keep functionality w/o Set
- Bump mocha to v4.

Reply:
The former is much better.

Reply:
I shall await @rogeliog's decision. (This was an async comment, cuz I _await_'ed in it. I hope my humor transcends the coldness of this comment box. :))

Reply:
I agree with @ljharb